packages/shared/src/components/cards/entity/SourceEntityCard.tsx

```diff
@@ -42,21 +48,25 @@ const SourceEntityCard = ({ source, className }: SourceEntityCardProps) => {
       }}
       entityName={name}
       actionButtons={
-        <>
-          <CustomFeedOptionsMenu
-            buttonVariant={ButtonVariant.Option}
-            className={{
-              menu: 'z-[9999]',
-            }}
-            {...menuProps}
-          />
-          <SourceActionsFollow
-            isFetching={false}
-            isSubscribed={isFollowing}
-            onClick={toggleFollow}
-            variant={ButtonVariant.Primary}
-          />
-        </>
+        !isLoading && (
```

Comment:
Why is this isLoading needed here to render or not?
As your description says something about if they follow or not.